src/lib/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ file(GENERATE
 
 if(SCORE_INSTALL_HEADERS)
   install(FILES
+        ${3RDPARTY_FOLDER}/QCodeEditor
```

Comment:
I don't think that this will work, did you check what gets copied if you make a build with 

     cmake <score folder> -GNinja -DSCORE_INSTALL_HEADERS=1 -DCMAKE_INSTALL_PREFIX=/tmp/installed -DDEPLOYMENT_BUILD=1 
     cmake --build .
     cmake --build . --target install
     
?

Reply:
ok so this works with the include ```${3RDPARTY_FOLDER}/QCodeEditor/include/internal/QCodeEditor.hpp```
I wonder if I need to include all of QCodeEditor files tho ?